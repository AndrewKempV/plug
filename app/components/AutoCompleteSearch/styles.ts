import { StyleSheet } from "react-native";
import { scale, verticalScale } from "react-native-size-matters/extend";
import { Colors, Layout } from "../../config/styles";
import Theme from "../../config/Theme";

const styles = StyleSheet.create({
  cancelSearch: {
    alignSelf: "center",
    justifyContent: "center",
    marginHorizontal: scale(16),
    position: "absolute",
    textAlign: "center"
  },
  currentLocationButton: {
    borderRadius: 8.5,
    flex: 1,
    height: 17,
    marginTop: 0,
    maxHeight: 17,
    maxWidth: 17,
    paddingTop: 0,
    width: 17
  },
  input: {
    backgroundColor: Colors.snow,
    color: Colors.black,
    flex: 1,
    paddingBottom: verticalScale(10),
    paddingLeft: scale(10),
    paddingRight: 0,
    paddingTop: verticalScale(10)
  },
  search: {
    flex: 1,
    flexDirection: "row",
    backgroundColor: Colors.niceBlue,
    maxHeight: "5%",
    borderRadius: 25,
    shadowColor: "#000",
    shadowOffset: {
      width: 0,
      height: 2
    },
    shadowOpacity: 0.23,
    shadowRadius: 2.62,

    elevation: 4
  },
  searchContainer: {
    borderBottomColor: "#00000033",
    borderRadius: 36,
    flex: 1,
    flexDirection: "row",
    maxHeight: "15%",
    position: "absolute"
  },
  searchIcon: {
    ...Layout.alignLeft,
    marginBottom: 0,

    marginLeft: 0,
    marginRight: 0,
    marginTop: 0,
    paddingHorizontal: scale(10),
    paddingVertical: 0
  },
  searchSection: {
    flex: 1,
    flexDirection: "row",
    ...Layout.alignLeft,

    backgroundColor: Colors.snow,
    borderRadius: 25,
    elevation: 4,
    marginHorizontal: scale(20),
    maxHeight: "5%",
    shadowColor: "#000",
    shadowOffset: {
      width: 0,
      height: 2
    },
    shadowOpacity: 0.23,
    shadowRadius: 2.62
  }
});

export default styles;
