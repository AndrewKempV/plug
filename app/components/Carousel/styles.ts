import { StyleSheet } from "react-native";
import { scale, verticalScale } from "react-native-size-matters/extend";

const styles = StyleSheet.create({
  container: {
    alignContent: "flex-start",
    flex: 1,
    justifyContent: "center",
    marginHorizontal: scale(100),
    maxWidth: scale(225)
  },
  imageContainer: {
    borderRadius: 23.1,
    height: 46.2,
    marginHorizontal: scale(20),
    width: 46.2
  },
  nextIcon: {
    marginRight: scale(30),
    marginTop: verticalScale(13),
    paddingLeft: scale(20)
  },
  socialIcon: {
    borderRadius: 23.1,
    height: 46.2,
    width: 46.2
  }
});

export default styles;
