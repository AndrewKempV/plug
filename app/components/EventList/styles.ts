import { StyleSheet } from "react-native";
import Metrics from "../../config/metrics";
import LayoutDebugger from "../../utils/LayoutDebugger";

const styles = StyleSheet.create({
  list: {
    marginLeft: Metrics.margin
  }
});

export default styles;
