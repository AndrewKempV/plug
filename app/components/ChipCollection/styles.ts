import { StyleSheet } from "react-native";
import { Layout } from "../../config/styles";
const styles = StyleSheet.create({
  chipContainer: {
    marginHorizontal: 10
  },
  rowContainer: {
    ...Layout.horizontalLeftAlign,
    marginVertical: 8.5
  }
});

export default styles;
