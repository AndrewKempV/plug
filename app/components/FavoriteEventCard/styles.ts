import { StyleSheet } from "react-native";
import Metrics from "../../config/metrics";
import { Colors, Fonts, Layout } from "../../config/styles";
import LayoutDebugger from "../../utils/LayoutDebugger";

const styles = StyleSheet.create({
  actionButtonContainer: {
    ...Layout.horizontalFlex,
    ...Layout.alignRight,
    height: 89,
    justifyContent: "flex-end"
  },
  container: {
    ...Layout.horizontalFlex,
    ...Layout.alignLeft,
    ...Layout.paddingDefault,
    height: 100,
    width: Metrics.DEVICE_WIDTH
  },
  dateLabel: {
    ...Layout.textLeft,
    color: Colors.darkBurgundy,
    fontFamily: Fonts.type.base,
    fontSize: 14,
    fontStyle: "normal",
    fontWeight: "500",
    paddingTop: 10,
    textAlign: "left",
    width: 155
  },
  eventInfoContainer: {
    ...Layout.verticalFlex,
    ...Layout.alignLeft,
    height: 63,
    paddingLeft: 16.7,
    width: 200
  },
  eventName: {
    ...Layout.textLeft,
    color: Colors.onyx,
    fontFamily: Fonts.type.medium,
    fontSize: 15,
    fontStyle: "normal",
    fontWeight: "500",
    width: 200
  },
  eventVenue: {
    ...Layout.textLeft,
    color: Colors.darkGrey,
    fontFamily: Fonts.type.base,
    fontSize: 14,
    fontStyle: "normal",
    fontWeight: "400",
    marginTop: 5,
    textAlign: "left",
    width: 200
  },
  favoriteIcon: {
    marginTop: 7,
    paddingRight: 16.2
  },
  image: {
    borderRadius: 8,
    height: 89,
    width: 102
  },
  shareIcon: {
    paddingBottom: 2,
    paddingRight: 12.2
  }
});
export default styles;
