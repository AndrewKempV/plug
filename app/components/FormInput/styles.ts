import { StyleSheet } from "react-native";
import Metrics from "../../config/metrics";
import { Colors } from "../../config/styles";

const styles = StyleSheet.create({
  formInput: {
    backgroundColor: Colors.whiteTwo,
    color: Colors.charcoalGrey,
    fontSize: 20,
    maxHeight: 57.1,
    textAlign: "center",
    width: Metrics.DEVICE_WIDTH
  }
});

export default styles;
