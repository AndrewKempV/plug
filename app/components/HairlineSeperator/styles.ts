import { StyleSheet } from "react-native";
import { scale, verticalScale } from "react-native-size-matters/extend";
import Fonts from "../../config/Fonts";
import Metrics from "../../config/metrics";
import { Colors } from "../../config/styles";

const styles = StyleSheet.create({
  centerText: {
    alignSelf: "center",
    color: Colors.charcoalGrey,
    fontFamily: "HelveticaNeue",
    fontSize: 14,
    fontWeight: "normal",
    paddingBottom: verticalScale(5),
    textAlign: "center"
  },
  hairline: {
    backgroundColor: Colors.charcoalGreyA100,
    height: 1,
    marginBottom: 0,
    marginLeft: scale(15.5),
    marginRight: scale(15.5),
    marginTop: 0,
    width: scale(347)
  },
  leftHairline: {
    backgroundColor: Colors.paleGrey,
    fontFamily: Fonts.type.bold,
    height: 3,
    marginLeft: scale(10),
    marginRight: scale(10),
    marginTop: verticalScale(7),
    width: scale(96)
  },
  rightHairline: {
    backgroundColor: Colors.paleGrey,
    fontFamily: Fonts.type.bold,
    height: 3,
    marginLeft: scale(10),
    marginRight: scale(10),
    marginTop: verticalScale(7),
    width: scale(96)
  },
  separatorContainer: {
    backgroundColor: Colors.whiteTwo,
    height: 1,
    width: Metrics.DEVICE_WIDTH
  },
  withTextSeparatorContainer: {
    flexDirection: "row",
    marginBottom: Metrics.marginHalf,
    marginHorizontal: scale(60),
    marginTop: Metrics.marginHalf
  }
});

export default styles;
