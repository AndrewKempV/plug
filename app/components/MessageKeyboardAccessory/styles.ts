import { StyleSheet } from "react-native";
import { scale, verticalScale } from "react-native-size-matters/extend";
import Fonts from "../../config/Fonts";
import Metrics from "../../config/metrics";
import { Colors } from "../../config/styles";
import { Layout } from "../../config/styles";

const styles = StyleSheet.create({
  accessoryViewContainer: {
    alignItems: "center",
    backgroundColor: Colors.snow,
    borderColor: Colors.paleGrey,
    borderStyle: "solid",
    borderWidth: 1,
    flexDirection: "column",
    height: verticalScale(160.7),
    maxHeight: verticalScale(160.7),
    padding: 8,
    width: scale(375)
  },
  addMediaButtonContainer: {
    backgroundColor: Colors.paleGrey,
    borderColor: Colors.burgundy,
    borderStyle: "solid",
    borderWidth: 3,
    height: 32,
    width: 32
  },
  messageButtonContainer: {
    ...Layout.alignCentered,
    backgroundColor: Colors.snow,
    borderColor: Colors.burgundy,
    borderRadius: 30,
    borderStyle: "solid",
    borderWidth: 3,
    height: 38,
    marginHorizontal: 5,
    width: 38
  },
  messageIcon: {
    paddingLeft: scale(0),
    paddingTop: verticalScale(14)
  },
  messagePanelDismissButton: {
    height: 32,
    marginLeft: 17,
    marginTop: 15.7,
    width: 32
  },
  messagePanelHeaderContainer: {
    ...Layout.alignLeft,
    ...Layout.horizontalFlex,
    height: 30,
    width: Metrics.DEVICE_WIDTH
  },
  messagePanelTitle: {
    ...Layout.textLeft,
    color: Colors.onyx,
    fontFamily: Fonts.type.base,
    fontSize: 18,
    fontStyle: "normal",
    fontWeight: "500",
    height: 22,
    marginLeft: 80,
    marginTop: 16.7,
    width: 117
  },
  textInput: {
    ...Layout.textLeftCenter,
    backgroundColor: Colors.paleGrey,
    fontSize: 16,
    height: 40,
    marginRight: 10,
    paddingLeft: 10,
    width: 200
  },
  textInputButton: {
    flexShrink: 1
  },
  textInputButtonLabel: {
    fontFamily: Fonts.type.base,
    fontSize: 15,
    fontStyle: "normal",
    fontWeight: "500",
    height: 18,
    width: 36,
    ...Layout.textLeft,
    color: Colors.burgundy
  },
  textInputContainer: {
    ...Layout.horizontalFlex,
    ...Layout.alignCentered,
    width: scale(345),
    height: verticalScale(73),
    borderRadius: 36,
    backgroundColor: Colors.paleGrey,
    borderStyle: "solid",
    borderWidth: 1,
    borderColor: "#c8d1d3",
    // padding: 10,
    marginLeft: 0,
    marginRight: 0
  }
});

export default styles;
