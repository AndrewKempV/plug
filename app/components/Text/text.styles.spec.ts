import { getStyleType } from "./text.styles";

describe("getStyleType", () => {
  it("should have cta styles", () => {
    const styles = getStyleType("cta");
    expect(styles).toMatchSnapshot();
  });
});
describe("getStyleType", () => {
  it("should have defailt styles", () => {
    const styles = getStyleType("default");
    expect(styles).toMatchSnapshot();
  });
});
describe("getStyleType", () => {
  it("should have error styles", () => {
    const styles = getStyleType("error");
    expect(styles).toMatchSnapshot();
  });
});
describe("getStyleType", () => {
  it("should have success styles", () => {
    const styles = getStyleType("success");
    expect(styles).toMatchSnapshot();
  });
});
describe("getStyleType", () => {
  it("should have B1 styles", () => {
    const styles = getStyleType("B1");
    expect(styles).toMatchSnapshot();
  });
});
describe("getStyleType", () => {
  it("should have B2 styles", () => {
    const styles = getStyleType("B2");
    expect(styles).toMatchSnapshot();
  });
});
describe("getStyleType", () => {
  it("should have H1 styles", () => {
    const styles = getStyleType("H1");
    expect(styles).toMatchSnapshot();
  });
});
describe("getStyleType", () => {
  it("should have H2 styles", () => {
    const styles = getStyleType("H2");
    expect(styles).toMatchSnapshot();
  });
});
describe("getStyleType", () => {
  it("should have H3 styles", () => {
    const styles = getStyleType("H3");
    expect(styles).toMatchSnapshot();
  });
});
describe("getStyleType", () => {
  it("should have H4 styles", () => {
    const styles = getStyleType("H4");
    expect(styles).toMatchSnapshot();
  });
});
describe("getStyleType", () => {
  it("should have H5 styles", () => {
    const styles = getStyleType("H5");
    expect(styles).toMatchSnapshot();
  });
});
