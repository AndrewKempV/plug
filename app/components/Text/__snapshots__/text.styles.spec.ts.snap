// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getStyleType should have B1 styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "fontSize": 16,
  "lineHeight": 22,
  "textAlign": "center",
}
`;

exports[`getStyleType should have B2 styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "fontSize": 12,
  "lineHeight": 16,
  "textAlign": "center",
}
`;

exports[`getStyleType should have H1 styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "fontSize": 30,
  "margin": 3,
  "textAlign": "center",
}
`;

exports[`getStyleType should have H2 styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "fontSize": 25,
  "margin": 3,
  "textAlign": "center",
}
`;

exports[`getStyleType should have H3 styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "fontSize": 22,
  "margin": 3,
  "textAlign": "center",
}
`;

exports[`getStyleType should have H4 styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "color": "#000000",
  "fontSize": 20,
  "lineHeight": 24,
  "textAlign": "center",
}
`;

exports[`getStyleType should have H5 styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "color": "#000000",
  "fontSize": 14,
  "lineHeight": 18,
  "textAlign": "center",
}
`;

exports[`getStyleType should have cta styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "color": "#6f0621",
  "fontSize": 17,
  "textAlign": "center",
}
`;

exports[`getStyleType should have defailt styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "color": "#000000",
  "fontSize": 17,
  "includeFontPadding": true,
  "textAlign": "center",
}
`;

exports[`getStyleType should have error styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "color": "#7c4d5a",
  "fontSize": 17,
  "textAlign": "center",
}
`;

exports[`getStyleType should have success styles 1`] = `
Object {
  "backgroundColor": "transparent",
  "color": "#800020",
  "fontSize": 17,
  "textAlign": "center",
}
`;
